header/verify.go

```diff
@@ -6,10 +6,25 @@ import (
 	"time"
 
 	"github.com/tendermint/tendermint/light"
+	"github.com/tendermint/tendermint/types"
 
 	libhead "github.com/celestiaorg/go-header"
 )
 
+// clockDrift defines how much new header's time can drift into
+// the future relative to the now time during verification.
+var clockDrift = 10 * time.Second
+
+// trustingPeriod is period through which we can trust a header's validators set.
+//
+// Should be significantly less than the unbonding period (e.g. unbonding
+// period = 3 weeks, trusting period = 2 weeks).
+//
+// More specifically, trusting period + time needed to check headers + time
+// needed to report and punish misbehavior should be less than the unbonding
+// period.
+var trustingPeriod = 168 * time.Hour
```

Comment:
Let's make it artificially low (or high) with the comment that it doesn't have an effect here and is handled in Syncer.

Having it here with a full description will cause more confusion.

Reply:
At this point, it does not seem to be worth using light.VerifyAdjacent tbh, let's revert it back